src/browser/modules/Sidebar/GuidesDrawer.tsx

```diff
@@ -59,33 +69,49 @@ function GuidesDrawer({
       ref={scrollRef}
     >
       <StyledGuidesDrawerHeader onClick={backToAllGuides}>
-        {!isDefaultGuide(guide) && (
+        {currentGuide !== null && (
           <BackIconContainer data-testid="guidesBackButton">
             <BackIcon width={16} />
           </BackIconContainer>
         )}
         Neo4j Browser Guides{' '}
       </StyledGuidesDrawerHeader>
       <StyledDrawerSeparator />
-      {!isDefaultGuide(guide) && (
-        <GuideTitle title={guide.title}>{guide.title}</GuideTitle>
+      {currentGuide === null ? (
+        <GuidePicker
+          remoteGuides={remoteGuides}
+          setGuide={setGuide}
+          updateRemoteGuides={updateRemoteGuides}
+        />
+      ) : (
+        <>
+          <GuideTitle title={currentGuide.title}>
+            {currentGuide.title}
+          </GuideTitle>
+          <GuideCarousel
+            slides={currentGuide.slides}
+            currentSlideIndex={currentGuide.currentSlide}
+            gotoSlide={gotoSlide}
+            scrollToTop={() =>
+              scrollRef.current?.scrollIntoView({ block: 'start' })
+            }
+          />
+        </>
       )}
-      <GuideCarousel
-        slides={guide.slides}
-        currentSlideIndex={guide.currentSlide}
-        gotoSlide={gotoSlide}
-        scrollToTop={() =>
-          scrollRef.current?.scrollIntoView({ block: 'start' })
-        }
-      />
     </StyledGuidesDrawer>
   )
 }
 
-const mapStateToProps = (state: GlobalState) => ({ guide: getGuide(state) })
-const mapDispatchToProps = (dispatch: any) => ({
-  backToAllGuides: () => dispatch(startGuide()),
-  gotoSlide: (slideIndex: number) => dispatch(gotoSlide(slideIndex))
+const mapStateToProps = (state: GlobalState) => ({
+  currentGuide: getCurrentGuide(state),
+  remoteGuides: getRemoteGuides(state)
+})
+const mapDispatchToProps = (dispatch: Dispatch<Action>) => ({
+  backToAllGuides: () => dispatch(resetGuide()),
+  gotoSlide: (slideIndex: number) => dispatch(gotoSlide(slideIndex)),
+  setGuide: (guide: Guide) => dispatch(setGuide(guide)),
```

Comment:
would be nice if we were consistent about naming of `setGuide` -> `setCurrentGuide` ...